### ISSUE TITLE
用語集：慣用句の見直し

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1728]
Derived https://github.com/waic/wcag21/issues/98

https://waic.jp/translations/WCAG21/#dfn-idioms
> 慣用句 (idiom)
> 慣用句は、単語単位で直接翻訳すると、その (文化的あるいは言語特有の) 意味が通じなくなる。

https://www.w3.org/TR/2018/REC-WCAG21-20180605/#dfn-idioms
> idioms cannot be translated directly, word for word, without losing their (cultural or language-dependent) meaning.

案：
慣用句は、(文化的又は言語依存の) 意味を失うことなく、逐語的に直接翻訳することはできない。

> "knocking over the beans" (豆をひっくり返す) や"spilling the vegetables" (野菜をこぼす) は同じ意味にはならない。
>
> However, "knocking over the beans" or "spilling the vegetables" does not mean the same thing.

案：
"knocking over the beans" (豆をひっくり返す) 又は "spilling the vegetables" (野菜をこぼす) は同じ意味にはならない。

**Comment by bakera** [comment]
>慣用句は、(文化的又は言語依存の) 意味を失うことなく、逐語的に直接翻訳することはできない。

これはちょっと多義的な表現になっているように感じます。「……なく、……できない」という文なので、「慣用句は意味を失わない、かつ直接翻訳できない」とも読めてしまいます。これは「慣用句は……失うことなく」を「慣用句を……失わせずに」とすることで緩和されるかもしれません。

「文化的又は言語依存」は「文化的である、または言語に依存する」とも、「文化的に依存する、または言語に依存する」ともとれますが、これはまあどちらでも大差ないですかね……? (おそらく前者)

細かいところで、「逐語的に直接」は directly, word for word, なので、原文の語順のままだと「直接、逐語的に」でしょうか (漢語の連続を嫌って順番を入れ替えた?)。これはまあどちらでも良いと思います。

たとえば、

- 慣用句を、その (文化的又は言語依存の) 意味を失わせずに、直接、逐語的に翻訳することはできない。

というところでしょうか。

**Comment by momdo** [comment]
慣用句「は」だと思うのですよね…これはどうでしょうか？

慣用句は、その (文化的又は言語依存の) 意味を失わせることなく、直接に、逐語的に翻訳することはできない。

**Comment by bakera** [comment]
良さそうに思います!
